### ISSUE TITLE
Item pages PDF viewer: Add "title" to iframe

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#142]
From a11y review: 
The frame that contains the PDF of the article on the Individual Results page does not have a name that is easily associated with the content. Add something more descriptive that alerts the screen reader user to the content within the frame. For example:

<iframe title="Article Preview" class="preview-iframe" id="preview-iframe" name="preview-iframe" width="100%" height="400" src="/records/mpp7x-hgg82/preview/Research%20Data%20Management%20Among%20Life%20Sciences%20Faculty.pdf">
</iframe>

**Comment by lhenze** [comment]
PR created: https://github.com/inveniosoftware/invenio-app-rdm/pull/1139
